From 7078cd58f7c8c17970b3679773ed6cf0d5679f3a Mon Sep 17 00:00:00 2001
From: "Yue, VincentX" <vincentx.yue@intel.com>
Date: Fri, 2 Mar 2018 10:43:13 +0800
Subject: [PATCH 11/16] Expand the NavigationBar buttons for the 1080P large
 display device

Due to  NavigationBar buttons are too smaller icon on the 1080P large
display device, lead to the CTS fail and the bad user experience.

Test: run cts -m CtsSystemUiTestCases -t android.systemui.cts.LightBarTests

Change-Id: I4b6db985c6abc3889fd041211579445af8961c67
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-58951
Signed-off-by: Yue, VincentX <vincentx.yue@intel.com>
---
 packages/SystemUI/res/layout/back.xml              |  4 ++--
 packages/SystemUI/res/layout/recent_apps.xml       |  4 ++--
 .../SystemUI/res/values-sw900dp-xxhdpi/dimens.xml  | 22 ++++++++++++++++++++++
 3 files changed, 26 insertions(+), 4 deletions(-)
 create mode 100644 packages/SystemUI/res/values-sw900dp-xxhdpi/dimens.xml

diff --git a/packages/SystemUI/res/layout/back.xml b/packages/SystemUI/res/layout/back.xml
index 43bec91..6843db9 100644
--- a/packages/SystemUI/res/layout/back.xml
+++ b/packages/SystemUI/res/layout/back.xml
@@ -24,8 +24,8 @@
     systemui:keyCode="4"
     android:scaleType="fitCenter"
     android:contentDescription="@string/accessibility_back"
-    android:paddingTop="15dp"
-    android:paddingBottom="15dp"
+    android:paddingTop="@dimen/home_padding"
+    android:paddingBottom="@dimen/home_padding"
     android:paddingStart="@dimen/navigation_key_padding"
     android:paddingEnd="@dimen/navigation_key_padding"
     />
diff --git a/packages/SystemUI/res/layout/recent_apps.xml b/packages/SystemUI/res/layout/recent_apps.xml
index c84d280..6b08cea 100644
--- a/packages/SystemUI/res/layout/recent_apps.xml
+++ b/packages/SystemUI/res/layout/recent_apps.xml
@@ -23,8 +23,8 @@
     android:layout_weight="0"
     android:scaleType="fitCenter"
     android:contentDescription="@string/accessibility_recent"
-    android:paddingTop="15dp"
-    android:paddingBottom="15dp"
+    android:paddingTop="@dimen/home_padding"
+    android:paddingBottom="@dimen/home_padding"
     android:paddingStart="@dimen/navigation_key_padding"
     android:paddingEnd="@dimen/navigation_key_padding"
     />
diff --git a/packages/SystemUI/res/values-sw900dp-xxhdpi/dimens.xml b/packages/SystemUI/res/values-sw900dp-xxhdpi/dimens.xml
new file mode 100644
index 0000000..abc6511
--- /dev/null
+++ b/packages/SystemUI/res/values-sw900dp-xxhdpi/dimens.xml
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ * Copyright (c) 2011, The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+-->
+<resources>
+    <!-- back button, Home button and recent button padding for sizing -->
+    <dimen name="home_padding">0dp</dimen>
+</resources>
+
-- 
1.9.1

